script.js

```diff
@@ -1,4 +1,180 @@
+      }
+      return (
+        "You win!💯💯<br> You chose " +
+        input +
+        ", while the computer chose ✌️ <br> You: " +
+        UserNumOfCorrectGuess +
+        " VS Computer: " +
+        CompNumOfCorrectGuess +
+        "<br> Now you can type scissors, paper or stone to play another round!✨"
+      );
+    }
+    if (input == "scissors") {
+      return (
+        "It's a draw...?👉👈 <br> You chose " +
+        input +
+        ", while the computer chose ✌️ <br> You: " +
+        UserNumOfCorrectGuess +
+        " VS Computer: " +
+        CompNumOfCorrectGuess +
+        "<br> Now you can type scissors, paper or stone to play another round!✨"
+      );
+    }
+    if (input == paper) {
+      CompNumOfCorrectGuess += 1;
+      console.log(CompNumOfCorrectGuess);
+    }
+    return (
+      "You lose...😧😧<br> You chose " +
+      input +
+      ", while the computer chose ✌️ <br> You: " +
+      UserNumOfCorrectGuess +
+      " VS Computer's score: " +
+      CompNumOfCorrectGuess +
+      "<br> Now you can type scissors, paper or stone to play another round!✨"
+    );
+  }
+
+  if (randomObject == 1) {
+    console.log(paper);
+    console.log("paper runs");
+    if (userVComputer(input, "✋")) {
+      if (input == scissors) {
+        UserNumOfCorrectGuess += 1;
+        console.log(UserNumOfCorrectGuess);
+      }
+      return (
+        "You win!💯💯<br> You chose " +
+        input +
+        ", while the computer chose ✋ <br> You: " +
+        UserNumOfCorrectGuess +
+        " VS Computer: " +
+        CompNumOfCorrectGuess +
+        "<br> Now you can type scissors, paper or stone to play another round!✨"
+      );
+    }
+
+    if (input == "paper") {
+      return (
+        "It's a draw...?👉👈<br> You chose " +
+        input +
+        ", while the computer chose ✋ <br> You: " +
+        UserNumOfCorrectGuess +
+        " VS Computer: " +
+        CompNumOfCorrectGuess +
+        "<br> Now you can type scissors, paper or stone to play another round!✨"
+      );
+    }
+
+    if (input == stone) {
+      CompNumOfCorrectGuess += 1;
+      console.log(CompNumOfCorrectGuess);
+    }
+    return (
+      "You lose...😧😧<br> You chose " +
+      input +
+      ", while the computer chose ✋ <br> You: " +
+      UserNumOfCorrectGuess +
+      " VS Computer: " +
+      CompNumOfCorrectGuess +
+      "<br> Now you can type scissors, paper or stone to play another round!✨"
+    );
+  }
+
+  if (randomObject != 0 || randomObject != 1) {
+    console.log(stone);
+    console.log("stone runs");
+    if (input == paper) {
+      UserNumOfCorrectGuess += 1;
+      console.log(UserNumOfCorrectGuess);
+    }
+    return (
+      "You win!💯💯<br> You chose " +
+      input +
+      ", while the computer chose ✊ <br> You: " +
+      UserNumOfCorrectGuess +
+      " VS Computer: " +
+      CompNumOfCorrectGuess +
+      "<br> Now you can type scissors, paper or stone to play another round!✨"
+    );
+  }
```

Comment:
Instead of having 3 large if statements inside the main function. You could abstract all this logic into another function to make your main function cleaner.